cmd/ctr/commands/run/run_windows.go

```diff
@@ -163,6 +163,11 @@ func NewContainer(ctx context.Context, client *containerd.Client, cliContext *cl
 			if idType == "" {
 				return nil, errors.New("devices must have a non-empty IDType")
 			}
+			parts := strings.Split(devID, ":")
+			if len(parts) > 3 {
+				return nil, errors.New("--device requires the format 'hostpath:containerpath:permission', while containerpath and permission are optional")
```

Comment:
Since containerpath and permission are never used, I recommend not taking those inputs from user even as optional args. Can you modify the error message also.

Reply:
remove this code, we use doc to prompt that it cannot be used in the window system.